File: cpp/1061.cpp
#include <iostream>
#include <stdio.h>
#include <string>

using namespace std;

int main()
{
    string tmp;
    int d1, h1, m1, s1, d2, h2, m2, s2;
    cin >> tmp >> d1 >> h1 >> tmp >> m1 >> tmp >> s1; 
    cin >> tmp >> d2 >> h2 >> tmp >> m2 >> tmp >> s2;

    if (s2 < s1) {
        s2 += 60;
        m2--;
    }

    if (m2 < m1) {
        m2 += 60;
        h2--;
    }

    if (h2 < h1) {
        h2 += 24;
        d2--;
    }

    printf("%d dia(s)\n", d2 - d1);
    printf("%d hora(s)\n", h2 - h1);
    printf("%d minuto(s)\n", m2 - m1);
    printf("%d segundo(s)\n", s2 - s1);
    return 0;
}

File: py/1061.py
days = [i for i in input().split()]
times = [int(i) for i in input().split(' : ')]
days.extend([i for i in input().split()])
times.extend([int(i) for i in input().split(' : ')])
    
days[1] = int(days[1])
days[3] = int(days[3])
    
if times[5] < times[2]:
    times[5] += 60
    times[4] -= 1

if times[4] < times[1]:
    times[4] += 60
    times[3] -= 1

if times[3] < times[0]:
    times[3] += 24
    days[3] -= 1
    
result = [
    days[3] - days[1], 
    times[3] - times[0], 
    times[4] - times[1], 
    times[5] - times[2]
    ]

print(f'{result[0]} dia(s)')
print(f'{result[1]} hora(s)')
print(f'{result[2]} minuto(s)')
print(f'{result[3]} segundo(s)')